docs/modules/ROOT/pages/reactive/authorization/method.adoc

```diff
@@ -304,11 +304,10 @@ and it will be invoked after the `@PostAuthorize` interceptor.
 
 == EnableReactiveMethodSecurity
 
-[WARNING]
+[INFO]
```

Comment:
Instead of changing the warning to info with the given message, I think we can simply remove the warning. Would you please consider this change?

Reply:
I made the changes as per your request if further changes have to be made please let me know

Reply:
Sorry, I don't think I was clear enough. 😓  I meant remove the note you added as well. The entire paragraph is not needed.

Reply:
oh my bad, I removed that para and also added "Supports reactive return types including Kotlin coroutines."